src/OpenTK/Platform/Linux/LinuxDisplayDriver.cs

```diff
@@ -383,11 +344,11 @@ private unsafe static DisplayResolution GetDisplayResolution(ModeInfo* mode)
             return null;
         }
 
-        public override bool TryChangeResolution(DisplayDevice device, DisplayResolution resolution)
+        public bool TryChangeResolution(object device, DisplayResolution resolution)
```

Comment:
Using object as an input for public API is not good. Coupled with perhaps having a base class for shared code and functionality, this base class should be used as the argument type instead.